uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-month.component.spec.js

```diff
@@ -109,31 +109,41 @@
           });
 
           describe("contacts' work pattern calendar", function () {
+            var calendarCallRecentsArgs;
+
+            beforeEach(function () {
+              calendarCallRecentsArgs = Calendar.get.calls.mostRecent().args;
```

Comment:
How is this even working? You are storing the args even before the function is called.

Reply:
I do not think so, there is a function call `sendShowMonthsSignal` in the parent `describe` that triggers the API call. `sendShowMonthsSignal` sends a signal and then digests the rootscope.

Reply:
Okay, I missed that part, Apologies.